libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -623,6 +629,24 @@ export const appStrings = {
     <UiString uiStringKey="titleDisplaySettingsSize">Text Size</UiString>
   ),
 
+  titleDisplaySettingsSensoryToggle: () => (
+    <UiString uiStringKey="titleDisplaySettingsSensoryToggle">
+      Accessibility Modes
+    </UiString>
+  ),
+
+  titleEnableAudioOnlyMode: () => (
```

Comment:
nit: for consistency, these should probably be `buttonEnableAudioOnlyMode`, `buttonExitAudioOnlyMode`

Reply:
Thanks. https://github.com/votingworks/vxsuite/pull/4518/commits/c035cbce847d1dd949e52a63a17865f2c8a2dd01 uses correct semantics for `button` and `title`